flink-table-store-hive/flink-table-store-hive-catalog/pom.xml

```diff
@@ -86,6 +86,12 @@ under the License.
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.jamon</groupId>
+            <artifactId>jamon-runtime</artifactId>
+            <version>2.4.1</version>
+        </dependency>
+
```

Comment:
I figured out in the other PR that we don't actually need to pin. Can you try without pinning (just the exclusions)?

Reply:
I removed that and my build succeeded locally. Thank you for the suggestion.